docs/sources/mimir/manage/tools/mimir-continuous-test.md

```diff
@@ -11,30 +11,32 @@ weight: 30
 
 # Grafana mimir-continuous-test
 
-As a developer, you can use the standalone mimir-continuous-test tool to run smoke tests on live Grafana Mimir clusters.
+As a developer, you can use the mimir-continuous-test tool to run smoke tests on live Grafana Mimir clusters.
 This tool identifies a class of bugs that could be difficult to spot during development.
 Two operating modes are supported:
 
 - As a continuously running deployment in your environment, mimir-continuous-test can be used to detect issues on a live Grafana Mimir cluster over time.
 - As an ad-hoc smoke test tool, mimir-continuous-test can be used to validate basic functionality after configuration changes are made to a Grafana Mimir cluster.
 
-## Download mimir-continuous-test
+## Download and Run mimir-continuous-test
 
 - Using Docker:
 
 ```bash
 docker pull "grafana/mimir-continuous-test:latest"
+docker run grafana/mimir-continuous-test
 ```
 
 - Using a local binary:
 
-Download the appropriate [mimir-continuous-test binary](https://github.com/grafana/mimir/releases/latest) for your operating system and architecture, and make it executable.
+Download the appropriate [mimir binary](https://github.com/grafana/mimir/releases/latest) for your operating system and architecture, and make it executable.
 
 For Linux with the AMD64 architecture, execute the following command:
 
 ```bash
-curl -Lo mimir-continuous-test https://github.com/grafana/mimir/releases/latest/download/mimir-continuous-test-linux-amd64
-chmod +x mimir-continuous-test
+curl -Lo mimir https://github.com/grafana/mimir/releases/latest/download/mimir-linux-amd64
+chmod +x mimir
+mimir -target=continuous-test
```

Comment:
Comes from the other PR, tried to follow the doc in order to tests the steps, but it failed, since latest release 1.12 doesn't support continuous-test target yet. 

`ts=2024-04-05T21:51:20.697771Z caller=log.go:103 level=error msg="error running application" err="selected target (continuous-test) is an internal module, which is not allowed"`

we could probably clarify what user should run depends on versions to avoid this.

Reply:
Do these docs update the website immediately or do they release along with the Mimir releases? Because this commit would be released along with 2.13 where the feature is included. 
<img width="574" alt="Screenshot 2024-04-05 at 4 36 07 PM" src="https://github.com/grafana/mimir/assets/6527685/f3cbc537-7094-4a82-ae97-9e5175f48c89">

Reply:
right I was thinking the md is used as doc, while the website should be updated when release.

Reply:
"next" docs are updated as this is merged to `main`. It's generally expected that the code and documentation should be in sync, of course with some flexibility.

The docs for released versions are only updated when a change is made to the release branch.